schemas/glean/events/events.1.parquetmr.txt

```diff
@@ -0,0 +1,183 @@
+        required binary key (UTF8);
+        required binary value (UTF8);
+      }
+    }
+    optional group number (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required double value;
+      }
+    }
+    optional group string_list (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required group value (LIST) {
+          repeated group list {
+            required binary element (UTF8);
+          }
+        }
+      }
+    }
+    optional group enumeration (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required binary value (UTF8);
+      }
+    }
+    optional group counter (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required int64 value;
+      }
+    }
+    optional group timespan (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required group value {
+          required int64 value;
+          required binary time_unit (UTF8);
+        }
+      }
+    }
+    optional group timing_distribution (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required group value {
+          required group range (TUPLE) {
+  required double lo;
+  required double hi;
+}
+required int64 bucket_count;
+optional binary histogram_type (UTF8);
+required group values (MAP) {
+  repeated group key_value {
+    required binary key (UTF8);
+    optional int64 value;
+  }
+}
+optional int64 underflow;
+optional int64 overflow;
+
+          required binary time_unit (UTF8);
+        }
+      }
+    }
+    optional group datetime (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required group value {
+          optional binary value (UTF8);
+          optional binary time_unit (UTF8);
+        }
+      }
+    }
+    optional group use_counter (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required group value {
+          optional group values (MAP) {
+            repeated group key_value {
+              required binary key (UTF8);
+              required int64 value;
+            }
+          }
+          optional group denominator {
+            optional binary name (UTF8);
+            optional int64 value;
+          }
+        }
+      }
+    }
+    optional group usage (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required boolean value;
+      }
+    }
+    optional group rate (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required int64 value;
+      }
+    }
+    optional group uuid (MAP) {
+      repeated group key_value {
+        required binary key (UTF8);
+        required binary value (UTF8);
+      }
+    }
+  }
+  optional group events (LIST) {
+    repeated group list {
+      required group element (TUPLE) {
+  required int64 timestamp;
```

Comment:
These issues should have been fixed with the recent parquetfmt changes, can you rebase and rebuild the schemas?

Reply:
Sure thing.  Should be done now.